### PR TITLE
Update QUEST_LV_0200.tsv (lines 7054,7061-7065,7070-7075,7077-7078,70…

### PR DESCRIPTION
**Pull request by fobm4ster**
…80,7082,7085-7087,7090,7092,7095)

Additional translations for the new content.
I was born in Korea and Korean is my first language, I manually translated all lines using my own knowledge. (Town names were either roughly translated, or pulled from lines previously translated)

Some KeyTerms:

에보니폰 : Ebonipone (This seems to be a mob name, and it cannot be "Ebony Pony" as someone has previously translated, because nowhere in the name does the term "Horse" appear,  It appears the original translation was thought to be "pony" due to phonetics)
라우키메 저습 : Rawookimae Wetlands.
민츠 대장님 : General Minchu (Not really sure how to phonetically translate the 츠 part of the name, mainly since its not a chu, or a che, but more of a cheh.  can be changed.

### CODE REVIEW

**Comment by Sunaries** [comment]
General Mintz?

**Comment by fobm4ster** [comment]
Yea I was kind of leaning towards that, but something about "Mintz" sounded really....lame lol.

**Comment by fobm4ster** [comment]
Also a thought for the "라우키메 저습"
Phonetically, it is "Rawooki", but then I just thought what if it is actually:
"Rocky Wetlands" ?
Just an idea if anyone else feels that way >.<

edit: wait nvm, just re-read it, its "Rawookimae" im going to update the phonetics on it, but I'm sure someone will come up with a proper name.

**Comment by Requinn** [comment]
I dont think itd be rocky since that would be like 로키 or something similar.

**Comment by fobm4ster** [comment]
kk then ill leave it at rawookimae for now

**Comment by Sunaries** [comment]
Laukime? I believe this is related to the Lithuanian language. Not sure, though.

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -7047,48 +7047,48 @@ QUEST_LV_0200_20150714_007050	You are aware that we're the unit sent out to take
...
 QUEST_LV_0200_20150714_007059	For the most part, do try and appease the monk's demands.{nl}We'll assist you wherever we can.
 QUEST_LV_0200_20150714_007060	I'm glad to hear that Natasha is doing well.{nl} I wish you the best at Raoki Swamps.
-QUEST_LV_0200_20150714_007061	
-QUEST_LV_0200_20150714_007062	
-QUEST_LV_0200_20150714_007063	
-QUEST_LV_0200_20150714_007064	
-QUEST_LV_0200_20150714_007065	
+QUEST_LV_0200_20150714_007061	Very well.  Did they finally restore the Monastery.{n1}Finally get to use my strength for once.
+QUEST_LV_0200_20150714_007062	Getting this far was all thanks to the Revelator.{n1}Now we can only hope that the Ebonipone is figured out.  Let us go.
+QUEST_LV_0200_20150714_007063	Senior Monk Fotos
```
Comment (Requinn) [inline QUEST_LV_0200.tsv:7059]: My previous PR had his name as Pothos. Could let IMC decide on that, though.

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**